app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt

```diff
@@ -45,7 +45,7 @@ fun MessageList(
 
       if (idx == messages.lastIndex) {
         suggestions.forEach { suggestion ->
-          SuggestionBox(suggestion, onSuggestionClick)
+          SuggestionBox(suggestion, onSuggestionClick, idx)
```

Comment:
This index is the index of the message on the whole list of messages. Isnt the analytics about the index of the suggestion inside the suggestion list itself ?